doc/source/whatsnew/v1.0.0.rst

```diff
@@ -3,10 +3,38 @@
 What's new in 1.0.0 (??)
 ------------------------
 
-.. warning::
-
-   Starting with the 0.25.x series of releases, pandas only supports Python 3.5.3 and higher.
-   See `Dropping Python 2.7 <https://pandas.pydata.org/pandas-docs/version/0.24/install.html#install-dropping-27>`_ for more details.
+New Deprecation Policy
+~~~~~~~~~~~~~~~~~~~~~~
+
+Starting with Pandas 1.0.0, pandas will adopt a version of `SemVer`_.
+
+Historically, pandas has used a "rolling" deprecation policy, with occasional
+outright breaking API changes. Where possible, we would deprecate the behavior
+we'd like to change, giving an option to adopt the new behavior (via a keyword
+or an alternative method), and issuing a warning for users of the old behavior.
+Sometimes, a deprecation was not possible, and we would make an outright API
+breaking change.
+
+We'll continue to *introduce* deprecations in major and minor releases (e.g.
+1.0.0, 1.1.0, ...). Those deprecations will be *enforced* in the next major
+release.
+
+Note that *behavior changes* and *API breaking changes* are not identical. If we
```

Comment:
First note here that API breaking changes will only happen in major releases? (as that part of the policy is actually not explicitly mentioned here)

Reply:
"here" being this section, or this document?

Reply:
I think it could be literally here (before the sentence that I commented on), but in the end it is mainly important it is mentioned in this document, if you see a better order. 

So the comment is basically that the part of "breaking changes will only happen in major releases" (which is explained in the policies.rst document) is not explicitly mentioned in the whatsnew note.

Reply:
Whoops, I was confused by the diff. Thought we were still in policies.st.